src/MessageHandler/Consumer.php

```diff
@@ -42,7 +42,12 @@ public function setNoAutoAck()
         $this->autoAck = false;
     }
 
-    public function getMessage(): ?MessageEnvelope
+    /**
+     * Get last MessageEnvelope
+     *
+     * @return MessageEnvelope|null
+     */
```

Comment:
this `phpDoc` is not required as it duplicates the `: ?MessageEnvelope` php7.1 syntax

therefore, I suggest to not add it, because it would be easy to forget to update it later